### ISSUE TITLE
Report enhancements made by JeanGarf / gnucash-android 

### ISSUE THREAD

**Issue by JeanGarf** [GnuCash-Pocket/gnucash-android#5]
#### Expected behaviour
Integrate JeanGarf's Pull Requests made in https://github.com/codinguser/gnucash-android/pulls, particularly :

- https://github.com/codinguser/gnucash-android/pull/882
- https://github.com/codinguser/gnucash-android/pull/895
- ...

#### Software specifications
* GnuCash Android version: 
* System Android version: 9
* Device type: Galaxy Note 8

**Comment by djbrown** [comment]
@JeanGarf do you think the project rename should wait until those major contributions are merged in?

**Comment by Kaligula0** [comment]
Does it fit XilinJia's commits? https://github.com/XilinJia/gnucash-android-kotlin I've read he made a lot od work recently, e.g. rewriting to Kotlin.

**Comment by Kaligula0** [comment]
I know XilinJia is claimed to be the GnuCash-Pocket developer, but he never asked for it and didn't say a word about that after he "got the job" – so maybe my comment above is pointless… 🤷

**Comment by Kaligula0** [comment]
I don't know the dis/advantages of moving to Kotlin. But, firsty, l I see that much of the work is done (thanks XilinJia!) and, secondly, I also commit to another Android app on GitHub where the developer recently moves constantly to Kotlin. And what I learned from quick googling – Kotlin got a language of choice when it comes to Android, so I guess Kotlin is more future-proof and will be easier to maintain. 🤔

**Comment by djbrown** [comment]
I think we should definitively migrate to Kotlin, especially since @XilinJia already did the work.
But I'm not sure in which order we should merge the contributions. Let's discuss that in #8

**Comment by JeanGarf** [comment]
I everybody,

I'm very pleased to see all these enthusiastics to restart the project !

I don't have lot of time at the moment and I don't know Kotlin, therefore It will be hard for me to contribute in the next months.

However, I would be pleased to answer you about my previous developments if you have questions when merging, about functionalities I developed.
I know I didn't do well about internationalization because I didn't know Crowdin, but all features work well on Samsung Galaxy Note 8.

See you...

**Comment by JeanGarf** [comment]
> @JeanGarf do you think the project rename should wait until those major contributions are merged in?

It would have been easier to merge before renaming, but now it is in Kotlin, so the merge will be difficult, renamed or not !

So now, I think it is not really the hardest problem !

**Comment by Kaligula0** [comment]
@JeanGarf, can any of your contributions be merged to this repo in its current state?

**Comment by JeanGarf** [comment]
@Kaligula0 Yes, my most recent commit is here 535f3d5 (https://github.com/JeanGarf/gnucash-android/tree/tw_develop) and contains all my contributions.
